SplunkAppForWazuh/appserver/static/js/controllers/settings/api/settingsApiCtrl.js

```diff
@@ -286,15 +282,15 @@ define(['../../module'], function(controllers) {
         } catch (err) {
           this.currentDataService
             .remove(id)
-            .then(() => {})
+            .then(() => { })
```

Comment:
Why here we are not using `await`? By the way, this method is pretty weird, try to re-implement it, please!